data-access-layer/iaas/GcpClient.js

```diff
@@ -58,7 +58,7 @@ class GcpClient extends BaseCloudClient {
       const options = {
         sourceSnapshot: `global/snapshots/${snapshotId}`,
         description: 'disk created via service fabrik',
-        type: opts.type || `projects/${this.settings.projectId || this.settings.credentials.project_id}/zones/${zone}/pd-ssd`,
+        type: `projects/${this.settings.projectId || this.settings.credentials.project_id}/zones/${zone}/diskTypes/${opts.type}`,
         labels: _.assign({}, opts.tags || {}, {
           createdBy: 'service-fabrik'
```

Comment:
Should be `createdby`, as keys should not contain capital letters. Commenting just so that it is not missed, we can handle it in other PR also.

Reply:
Yes- will change this in a new PR